distance_validation.py

```diff
@@ -57,14 +73,26 @@ def sample(num, low, high, distance):
 
         user_one_attributes = user_data.iloc[user_one_id].values[1:num_col+1] #should be [1:8] but sample data is missing a column
         user_two_attributes = user_data.iloc[user_two_id].values[1:num_col+1]
+        dist = distance(user_one_attributes, user_two_attributes)
+
+        random_sample = random_sample.append({"user_one_id": user_one_id,
+                                            "user_two_id": user_two_id,
+                                            "distance": dist},
+
         l2 = distance(user_one_attributes, user_two_attributes)
```

Comment:
Should this line be removed? It is only l2 if the distance function passed in is euclidean distance.